### DISCUSSION TITLE
Allow for ip as prefix instead only suffix

### DISCUSSION THREAD

**Discussion by jpambrun-vida** [pyrou/traefik.me#7] [General]
1-1-1-1-www.traefik.me resolve to 127.0.0.1, while www-1-1-1-1.traefik.me resolves to 1.1.1.1. 

1-1-1-1-www.traefik.me looks cleaner to me, and is easier to use with HostRegexp(`^.*www.traefik\\.me$`).

Also the [privkey.pem](https://traefik.me/#) link is not pointing to the right location.

**Comment by pyrou** [comment]
Nice idea, I'll implement IP as prefix next week.

About privkey link, I was facing so much certificate revocation (made by people that probably didn't understood the purpose of exposing these private keys) that I decided to add a small disclaimer in a JS alert

**Reply by jpambrun-vida** [reply]
Interesting. I made a similar tool in JS [here](https://github.com/jpambrun/dnsssl). I remember people feeling very strongly about private keys as well.. I ended up hiding it behind a trivial token.. I was wondering if you had any heat. Thanks for sharing.

**Reply by pyrou** [reply]
I first thought about requiring a free registration to get the cert link

But this would have slowed down adoption. And also the wget in docker-compose trick wouldn't be that magic with an API key or similar.

I just started working on the IP as prefix ;) expect to see it in a few ~days~ hours 😅🏖️

**Comment by pyrou** [comment]
```
dig A 123.123.123.123.foo.bar.traefik.me +noall +answer

; <<>> DiG 9.11.3-1ubuntu1.17-Ubuntu <<>> A 123.123.123.123.foo.bar.traefik.me +noall +answer
;; global options: +cmd
123.123.123.123.foo.bar.traefik.me. 3600 IN A	123.123.123.123
```
